app/src/main/res/layout/fragment_tracking_protection_blocking.xml

```diff
@@ -14,7 +14,7 @@
         <TextView
             android:id="@+id/details_blocking_header"
             style="@style/QuickSettingsText"
-            android:fontFamily="@font/metropolis_bold"
+            app:fontWeight="@integer/font_weight_bold"
```

Comment:
I don't see the fontWeight doing anything here. Perhaps we need to set the textStyle to bold for all of metropolis_bold.